src/Sylius/Bundle/WebBundle/Resources/views/Frontend/Cart/summary.html.twig

```diff
@@ -43,10 +42,11 @@
         {% endfor %}
     </tbody>
     <tfoot>
-        {% if cart.adjustmentsTotal(promotionAdjustment) %}
+        {% set promotionTotal = cart.getPromotionsTotalRecursively() %}
```

Comment:
We should display promotion total that has Order + Order Item discounts included. Right now it only displays discount on Order level